pkg/schedule/schedulers/evict_leader.go

```diff
@@ -385,6 +378,22 @@ func scheduleEvictLeaderOnce(r *rand.Rand, name string, cluster sche.SchedulerCl
 	return ops
 }
 
+func createOperatorWithSort(name string, cluster sche.SchedulerCluster, candidates *filter.StoreCandidates, region *core.RegionInfo) (*operator.Operator, error) {
```

Comment:
How about renaming it to `CreateTransferLeaderOperatorToLowestScoreStore` and moving it to operator.go?